normalize_repos/normalize_repos.py

```diff
@@ -54,7 +54,7 @@
     ],
     "vocabulary": [
         "ci/circleci: lint",
-        "ci/circleci: test",
+        "ci/circleci: unit",
         "ci/circleci: build",
```

Comment:
Since we're planning to drop CircleCI altogether from Vocabulary, it would be better if you could replace all the `ci/circleci` jobs with the jobs from our [GitHub Actions configuration file](https://github.com/creativecommons/vocabulary/blob/master/.github/workflows/ci.yml).

Reply:
@dhruvkb Is ```ci/vocabulary-ci```  fine?

Reply:
I don't know. @TimidRobot might.

Reply:
The check names are visible on each PR. e.g. https://github.com/creativecommons/vocabulary/pull/302

I think it's all the `vocabulary-ci/` checks.